src/mdns/mdns_mdnssd.cpp

```diff
@@ -333,42 +319,24 @@ void PublisherMDnsSd::Process(const MainloopContext &aMainloop)
         }
     }
 
-    for (auto &subscription : mSubscribedServices)
+    for (DNSServiceRef serviceRef : readyServices)
```

Comment:
```suggestion
    for (const DNSServiceRef &serviceRef : readyServices)
```

Reply:
Actually this is not change of this PR.

Since `DNSServiceRef` is already a pointer, no need to use `const &`.